ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/EeaPrivateNonceProvider.java

```diff
@@ -12,66 +12,42 @@
  */
 package tech.pegasys.ethsigner.core.requesthandler.sendtransaction.transaction;
 
+import tech.pegasys.ethsigner.core.jsonrpc.JsonRpcRequest;
 import tech.pegasys.ethsigner.core.requesthandler.sendtransaction.NonceProvider;
 
-import java.io.IOException;
 import java.math.BigInteger;
 import java.util.List;
 
-import com.google.common.collect.Lists;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.web3j.protocol.Web3jService;
-import org.web3j.protocol.core.Request;
-import org.web3j.protocol.core.methods.response.EthGetTransactionCount;
 import org.web3j.utils.Base64String;
 
 public class EeaPrivateNonceProvider implements NonceProvider {
 
-  private static final Logger LOG = LogManager.getLogger();
-
-  private final Web3jService web3jService;
   private final String accountAddress;
   private final Base64String privateFrom;
   private final List<Base64String> privateFor;
+  private VertxNonceRequestTransmitter vertxNonceRequestTransmitter;
```

Comment:
can be declared as `final` since initialised via constructor?

Reply:
done in later PR.